tests/utils/custom.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "JSON_P1",
+  "title": "JSON Tailored Profile P1",
+  "base_profile_id": "P1",
+  "rules": {
+    "R1": {
+      "selected": false
+    },
+    "R3": {
+      "selected": true,
+      "role": "unchecked",
+      "severity": "unknown"
+    }
+  },
+  "variables": {
+    "V1": {
+      "value": "New Value"
+    },
+    "V2": {
+      "select": "some"
```

Comment:
What is `select` in the context of variables?

Reply:
In XCCDF it is a selector ID. For a Value with option `<... selector="some">100</>` it will set the final value to 100 for `{"V2": {"select": "some"}}`.

Reply:
In XCCDF Tailoring it is rendered as `<value-refine>`.

Reply:
Considering we are mirroring XCCDF attributes, why not to use `selector` instead?

Reply:
I prefer `selector` or `variable_selector`

Reply:
"base_profile_id" `additionalProperties`, should we stick to a single case format?

Reply:
"Selector" sounds a better name than "select" because it suggests what should be the value of the attribute.

Reply:
Okay, after reading around, I propose this:
```json
{
  "id": "tailored_profile",
  "title": "Test",
  "profile_id": "original_profile",
  "rules": {
    "some_rule_id_here": {
      "evaluate": true
    },
    "different_rule_id_here": {
      "role": "unscored",
      "severity": "high"
    }
  },
  "variables": {
    "a_variable_id": {
      "value": "New Value"
    },
    "3rd_variable_id": {
      "option_id": "var3_value_100"
    }
  }
}
```
Reasoning:
- **base_profile_id -> profile_id**: We tailor profiles, and tailoring becomes a new profile. There is no reasonable way to join profiles together (and unlikely there will ever be) or use an additional profile in this context. Let's keep it simple.
- **selected -> evaluate**: It directly conveys the message that the rule will be forcibly evaluated (with tailoring). Better that select (select to do what?).
- **select -> option_id**: Why not _selector_? It is a very XCCDF-specific term, that has more functions and meaning than just a reference to a predefined value (it could be used to orchestrate rules/clusters behavior as well). On the other hand, every XCCDF Value with predefined values has them in _options_ (pretty common name in UI/UX and CS in general), so we are either set the value or refer to a value using option identifier.

Reply:
Not sure if we should have `"severity": "HIGH"` instead of `"severity": "high"` to stress the fact that it is a ENUMERATION_CONSTANT.

Reply:
> We kinda trying the opposite: abstracting XCCDF implementation and terminology.

What do you mean, I don't get it.

Reply:
> "base_profile_id" `additionalProperties`, should we stick to a single case format?

What do you mean, I don't get it.

Reply:
Thank you for your valuable input, gentlemen. I've update the PR.